randomizer/Patching/CosmeticColors.py

```diff
@@ -2532,7 +2557,18 @@ def placeKrushaHead(slot):
     kong_face_textures = [[0x27C, 0x27B], [0x279, 0x27A], [0x277, 0x278], [0x276, 0x275], [0x273, 0x274]]
     unc_face_textures = [[579, 586], [580, 587], [581, 588], [582, 589], [577, 578]]
     ROM_COPY = LocalROM()
-    ROM_COPY.seek(0x1FF6000)
+    match slot:
```

Comment:
Replace this with a `getImageFile()` call. Use `getBonusSkinOffset()` to get the file index that's needing to be used for the file index argument. You'll then want to change the BinaryIO reads to reading from a bytearray

Reply:
Update, had a little more of a think on this. getImageFile returns a PIL Image rather than a bytes object. So ignore the BinaryIO segment that. use `writeColorImageToROM()` instead